math/pronicnumber_test.go

```diff
@@ -0,0 +1,436 @@
+// pronicnumber_test.go
+// description: Returns true if number is pronic, false otherwise
+// author: Akshay Dubey (https://github.com/itsAkshayDubey)
+// see pronicnumber.go
+
+package math_test
+
+import (
+	"testing"
+
+	"github.com/TheAlgorithms/Go/math"
+)
+
+func TestPronicNumber(t *testing.T) {
+	var tests = []struct {
+		name          string
```

Comment:
The `name` seems to be redundant in this structure. One can save lots of bytes by _generating_ the test name using `n`.

Reply:
I don't this that is required. This is fine to do IMO.